File: 75-sort-colors/sort-colors.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) {
        int cred = 0;
        int cblue  = 0;
        int cwhite = 0;
        for (int i = 0 ; i< nums.size() ; i++){
            if(nums[i]==0){
                cred+=1;
            }
            else if (nums[i]==1){
                cwhite+=1;
            }
            else{
                cblue+=1;
            }
        }
         cwhite = cred+cwhite;
         cblue = cwhite+cblue;
        for(int i = 0 ; i<cred ; i++){
            nums[i]=0;
        }
        for (int i = cred; i<cwhite;i++){
            nums[i]=1;
        }
        for (int i = cwhite ; i<cblue ; i++){
            nums[i] = 2;
        }
        
        


   }
};

File: 75-sort-colors/sort-colors.java
class Solution {
    public void sortColors(int[] nums) {
        int red = 0;
        int white = 1;
        int blue = 2;
        int i = 0;
        int j = 0;
        int k = 0;
        for (int m = 0 ; m<nums.length ; m++)
        {
            if(nums[m]==0)
            {
                i++;
            }
            else if (nums[m]==1)
            {
                j++;
            }
            else
            {
                k++;
            }
        }
        // int q = i;
        // int r =i+j;
        int x = 0;
        while(i>0)
        {
            nums[x]= 0;
            x++;
            i--;
        }
        while(j>0)
        {
            nums[x] = 1;
            x++;
            j--;
        }
        while(k>0)
        {
            nums[x] = 2;
            x++;
            k--;
        }
        
    }
}